api.bs

```diff
@@ -1214,12 +1214,17 @@ To perform <dfn>common matching logic</dfn>, given
     1.  If |now| - |lookbackDays| is after |impression|'s [=impression/timestamp=],
         [=iteration/continue=].
 
+    1.  If |impression|'s [=impression/conversion sites=] [=set/is empty|is not empty=]
```

Comment:
Isn't it impossible to have an empty set?

Reply:
No, this PR currently allows there to be an empty set of conversion sites, unless I made a mistake somewhere.